packages/neon/neon_files/lib/widgets/actions.dart

```diff
@@ -64,15 +65,23 @@ class FileActions extends StatelessWidget {
         );
         b.dispose();
         if (result != null) {
-          bloc.move(details.path, result..add(details.name));
+          bloc.move(
+            details.path,
+            PathUri(
+              pathSegments: [
+                ...result.pathSegments,
+                details.path.basename,
+              ],
+            ),
```

Comment:
We quite often join a path that is a directory with a file name.
I think this could be a helper method in the PathUri that also asserts that the original path is a directory.

Reply:
Yeah I thought about this and didn't find a good solution yet